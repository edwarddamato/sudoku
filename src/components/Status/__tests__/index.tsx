import * as React from 'react';
import { Status } from '../index';
import { create } from 'react-test-renderer';

it('renders correctly', () => {
  const status = create(
    <Status />
  ).toJSON();
  expect(status).toMatchSnapshot();
});