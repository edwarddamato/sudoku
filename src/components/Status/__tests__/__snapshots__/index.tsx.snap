// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="component_status"
>
  <ul
    className="status_list"
  >
    <li
      className="status_list-item"
    >
      <div
        className="status_item-label"
      >
        Current Cell Index
      </div>
      <div
        className="status_item-value"
      >
        -1
      </div>
    </li>
    <li
      className="status_list-item"
    >
      <div
        className="status_item-label"
      >
        Current Block
      </div>
      <div
        className="status_item-value"
      >
        -1
      </div>
    </li>
    <li
      className="status_list-item"
    >
      <div
        className="status_item-label"
      >
        Current Coords
      </div>
      <div
        className="status_item-value"
      >
        -1, -1
      </div>
    </li>
  </ul>
</div>
`;
